src/piadm.sh

```diff
@@ -954,6 +954,25 @@ update_CN() {
 	update_boot_sectors "$pool" "$bootfs"
 }
 
+declare stick_already_mounted
+
+# Wrapper around `sdc-usbkey mount`. Preserves was-already-mounted (or not).
+usb_mount_if_not_already() {
+	status=$(sdc-usbkey status)
+	if [[ "$status" == "mounted" ]]; then
+		stick_already_mounted="yes"
+	fi
+	sdc-usbkey mount
```

Comment:
I assume we're still executing it here because we want to capture the returned path?